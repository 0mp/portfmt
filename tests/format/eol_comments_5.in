LDFLAGS+=	-Wl,-E # plugins
LDFLAGS+=	-Wl,--as-needed # ICE, SM, Xext
