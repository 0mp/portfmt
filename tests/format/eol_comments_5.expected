# plugins
LDFLAGS+=	-fstack-protector-strong -Wl,-E
# ICE, SM, Xext
LDFLAGS+=	-Wl,--as-needed
